pubsub/src/main/java/com/github/twitch4j/pubsub/TwitchPubSub.java

```diff
@@ -718,6 +720,13 @@ protected void onTextMessage(String text) {
                     }
                 } else if ("radio-events-v1".equals(topicName)) {
                     eventManager.publish(new RadioEvent(TypeConvert.jsonToObject(rawMessage, RadioData.class)));
+                } else if ("shoutout".equals(topicName)) {
+                    if ("create".equalsIgnoreCase(type)) {
+                        CreateShoutoutData shoutoutInfo = TypeConvert.convertValue(msgData, CreateShoutoutData.class);
+                        eventManager.publish(new CreateShoutoutEvent(lastTopicIdentifier, shoutoutInfo));
```

Comment:
could we name this `ShoutoutCreatedEvent`?

Thats the same name pattern we used for other pubsub events with `Created`, `Updated`, `Deleted` at the end of the class.

Reply:
Good point! Fixed in https://github.com/twitch4j/twitch4j/pull/660/commits/8a1645be49fb00ce097f1149ed3ada0ef21d805b